jina/peapods/zmq.py

```diff
@@ -713,12 +714,11 @@ def _check_msg_version(msg: 'jina_pb2.Message'):
                                 'the message is probably sent from a very outdated JINA version')
 
 
-def _extract_bytes_from_msg(msg: 'jina_pb2.Message') -> Tuple:
+def _extract_bytes_from_msg(docs: Iterable['jina_pb2.Document']) -> Tuple:
```

Comment:
That should be now `_extract_bytes_from_docs`, no? Why moving the document extraction outside of this method?

Reply:
Ah I see, for testability. :+1:

Reply:
for testability yes, but I am with u in changing the name, let's do it